src/core/Microsoft.PowerToys.Settings.UI.Lib/EnabledModules.cs

```diff
@@ -21,13 +21,13 @@ public EnabledModules()
         [JsonPropertyName("FancyZones")]
         public bool FancyZones
         {
-            get => this.fancyZones;
+            get => fancyZones;
             set
             {
-                if (this.fancyZones != value)
+                if (fancyZones != value)
                 {
                     LogTelemetryEvent(value);
-                    this.fancyZones = value;
+                    fancyZones = value;
```

Comment:
@crutkas Do these throw errors?  I thought using 'this.privatemembervariable' was good practice as far a stylecop was concerned.

Reply:
FYI this is the rule I was referring to.  Did we disable this? [SA1101](https://documentation.help/StyleCop/SA1101.html)

Reply:
NVM Should have looked at our GlobalSupression file first. 
[assembly: SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1101:PrefixLocalCallsWithThis", Justification = "We follow the C# Core Coding Style which avoids using `this` unless absolutely necessary.")]